src/liftwing_api/models/revertrisk.py

```diff
@@ -0,0 +1,33 @@
+import requests
+import json
+from liftwing_api.models.liftwing_model import LiftwingModel
+from typing import Dict
+
+class RevertRiskAPIModel(LiftwingModel):
+    def __init__(self, base_url="https://api.wikimedia.org/service/lw/inference/v1/models/revertrisk-language-agnostic:predict"):
+        super().__init__(base_url)
+
+    def request(self, payload: Dict[str, int], method: str = "POST", headers: Dict[str, str] = None) -> Dict[str, int]:
+        """
+        This function makes a POST request to https://api.wikimedia.org/service/lw/inference/v1/models/revertrisk-language-agnostic:predict
+        using the language parameter and returns a JSON
+        language is for the different wiki languages, rev_id is the specific revisions
+        """
+        language = payload.get("lang")
+        if language is None:
+            raise ValueError("'lang' parameter is required in the payload.")
+        rev_id = payload.get("rev_id")
+        if rev_id is None:
+            raise ValueError("rev id is none, add revision id to continue")
+
+        if headers is None:
+            headers = {}
+
+        response = requests.post(self.base_url, json=payload, headers=headers)
+
+        if response.status_code == 200:
+            return response.json()
+        else:
+            response.status_code == 400
+            raise ValueError(f"Unexpected error occurred: {response.status_code}")
```

Comment:
Manually changing the status_code for the response wouldn't be a good practice as it may end up being misleading in case we want to debug or understand what went wrong. 
For example in the case where we get a 5xx response caused by the server we would want to look at what is wrong with the service itself while a 400 error usually means `Bad input` so we would end up going in the wrong way (checking our input, reading API documentation etc).

Fo now I would recommend that we just return the response.json() in both cases and we can look at what to do with failing requests in the future.
```suggestion
        return response.json()        
```
feel free to test it and let me know if you have a different opinion!

Reply:
updated